src/app/project/ontology/property-form/property-form.component.spec.ts

```diff
@@ -190,16 +207,48 @@ class ListHostComponent {
 
 }
 
-@Component({ selector: 'app-string-literal-input', template: '' })
```

Comment:
In https://github.com/dasch-swiss/dsp-app/pull/893 we spent a lot of time decluttering the unit test logs. This mock component should remain as without it we get spam in the logs again.

<img width="897" alt="Screen Shot 2023-02-14 at 08 57 30" src="https://user-images.githubusercontent.com/60604010/218677687-07d1056c-8a7e-455e-96ff-f0812e720ea9.png">